src/App.js

```diff
@@ -1,10 +1,37 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import Header from './components/Header';
 import './App.css';
+import Missions from './pages/Missions/Missions';
+import Profile from './pages/Profile/Profile';
+import Dragons from './pages/Dragons/Dragons';
+import Rockets from './pages/Rockets/Rockets';
+import { getMissions } from './redux/missions/missions';
+import { getDragons } from './redux/dragons/dragons';
+import { getRockets } from './redux/rockets/rockets';
 
-const App = () => (
-  <div>
-    Hello React
-  </div>
-);
+const App = () => {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(getMissions());
+    dispatch(getDragons());
+    dispatch(getRockets());
+  }, [dispatch]);
```

Comment:
- [ ] For each module, the view is currently showing only the loading status or the fetched results but what if the server didn't respond with data? For this case, I'd suggest that you guys conditionally render the `Rockets`, `Missions`, and `Dragons` to show a disclaimer if the server failed to respond or didn't send data at all.

Reply:
Hi @omar-labana! Thank you for pointing this out. We have implemented this change.